spec/requests/personas_spec.rb

```diff
@@ -1,10 +1,33 @@
 require "rails_helper"
 
 RSpec.describe "Personas", type: :request do
-  describe "GET /personas" do
-    it "returns http success" do
-      get personas_path
-      expect(response).to have_http_status(:success)
+  context "placements" do
```

Comment:
I thought we are not doing these request specs for controller that are not API as per https://github.com/DFE-Digital/itt-mentor-services/pull/34

Reply:
You are right of course! I will delete it.